app/javascript/stylesheets/application.css

```diff
@@ -1,3 +1,3 @@
 body {
-  color: red;
+    color: red;
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/noboru-i/sample-rails-ruby/blob/33b3874ad4a7137960c5bf6b0dfa25cc039be9cc/app/javascript/stylesheets/application.css#L2"></span>Expected indentation of 2 spaces (indentation)</td>
    </tr>
  </tbody>
</table>